assets/utils/h5p.js

```diff
@@ -989,11 +990,21 @@ function itemUntranslatable(property, value, parent) {
       return false;
       break;
     case 'default':
-
+      if (typeof value !== 'string') {
+        return true;
+      }
       if (parent.type === 'select' || parent.widget === 'colorSelector') {
         return true;
       }
-
+      if (!value.replaceAll(new RegExp(/<\/?[a-z][^>]*>/ig), '')) {
+        return true;
+      }
+      if (new RegExp(/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/).test(value) === true || ['rgb(', 'hsv '].indexOf(value.substr(0, 4)) !== -1) { // color codes
+        return true;
+      }
+      if (languageCodes.indexOf(value.toLowerCase()) !== -1) { // language codes
+        return true;
+      }
       switch (typeof(value)) {
```

Comment:
I think that this switch can also be removed now, since we're added a condition to allow only strings anyway